.github/workflows/tests.yml

```diff
@@ -0,0 +1,132 @@
+name: Tests
+on: [push]
+jobs:
+  e2e_1:
```

Comment:
I duplicated the 5 steps to make sure the e2e tests will run as often as possible in the next few days.
This will allow us to seek and destroy flakiness in tests much faster.